src/components/SQForm/useFormButton.tsx

```diff
@@ -12,11 +12,12 @@ type UseFormButtonReturnType = {
   isValid: boolean;
   handleClick: DebouncedState<React.MouseEventHandler<HTMLButtonElement>>;
   [key: string]: unknown;
+  handleReset: () => void;
```

Comment:
This `handleReset` should actually come from the `...rest` in `useFormikContext`. I'll be creating an issue and PR for that and a couple other type improvements soon

Reply:
#382

Reply:
Merged #382, so the handleReset should be typed properly and you can remove this addition to useFormButton